src/en/news/blog/2025/user-accounts-dashboard/index.md

```diff
@@ -0,0 +1,155 @@
+- **Shared visibility**: Every user or role in the account can list and manage account‑owned buckets.
+- **Streamlined management**: Apply IAM policies centrally within the account for consistent permissions.
+
+---
+
+## 2. Roles & Responsibilities: Understanding Account Users
+
+| Role Type | What It Is | Permissions |
+|-------------------|----------------------------------------------|--------------------------------------------------------------------|
+| **Account Root User** | The master account owner, created by an admin | Full default access to account’s resources; used to define policies and create sub-users |
+| **IAM Users / Roles** | Scoped identities created by the root user | No access by default—they need explicit IAM policies to perform actions |
+
+
+## 3. User Accounts Management
+### 3.1 Account Creation
+
+The account feature is added under dashboard menu Object > Accounts. Once navigated to this page it will show the list of accounts.
+
+![Step 1: Account list page](images/account-list.png)  
+    *Figure: Step 1 - Navigating to Accounts page*
+
+Now we will click on the `Create` button for creating an account.
+
+We will be getting a form to enter values for following fields:
+
+![Step 2.1: Create Account: Enter account details](images/enter-account-form-details1.png)  
+    *Figure: Step 2.1 - Enter account details*
+
+![Step 2.2: Create Account: Enter quota details](images/enter-account-form-details2.png)  
+    *Figure: Step 2.2 - Enter quota details*
+
+After entering the details click on create Account button, post successful account creation you will be redirected to the accounts list page in which we can see the newly created account listed in table, and on expanding the table row, we can see the account details which shows the Account Quota & Bucket Quota for the account.
+
+![Step 3: Account list & details](images/account-details.png)  
+    *Figure: Step 3 - Account details*
+
+### 3.2 Link user with an Account
+
+After creating an account, the next step would be to create a account user. Lets see how its done.
+
+Navigate to the user page and click on `Create` button for creating a user.
+
+An account user can be created by two ways:  
+ - Creating a new user and linking it with an account at the same time.
+ - If you already have a user created then that user can be edited, and can be linked to an account.
+
+You will be seeing the user form in which `Link Account` will be the first input selector from which you can choose to link the user with an account, from the account list.
+
+> **NOTE:** This option would be visible only if you have accounts created.  
+
+![Step 4.1: Create User: Link user with account](images/link-user-with-account.png)  
+*Figure: Step 4.1 - Link user with account*
+
+Also, as seen once the account is selected, there will be another option to make it as a `root account user`. We can select the checkbox if this user needs to have additional permission to manage/create other user.
```

Comment:
Also, as seen, once the account is selected, another option will appear to mark the user as a root account user. You can select this checkbox if the user needs additional permissions to manage or create other users.